cmd/nada-backend/main.go

```diff
@@ -225,3 +231,40 @@ func runMetabase(ctx context.Context, log *logrus.Entry, cfg Config, repo *datab
 	go metabase.Run(ctx, MetabaseUpdateFrequency)
 	return nil
 }
+
+func isLeader() (bool, error) {
+	electorPath := os.Getenv("ELECTOR_PATH")
+	if electorPath == "" {
+		// local development
+		return true, nil
+	}
+
+	hostname, err := os.Hostname()
+	if err != nil {
+		return false, err
+	}
+
+	resp, err := http.Get("http://" + electorPath)
+	if err != nil {
+		return false, err
+	}
+	defer resp.Body.Close()
+	bodyBytes, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return false, err
+	}
+
+	var electorResponse struct {
+		Name string
+	}
+
+	if err := json.Unmarshal(bodyBytes, &electorResponse); err != nil {
+		return false, err
+	}
+
+	if hostname == electorResponse.Name {
+		return true, nil
+	}
+
+	return false, nil
```

Comment:
```suggestion
	return hostname == electorResponse.Name, nil
```

Du kan forenkle slutten til dette.